airbyte-cdk/python/connector_builder/main.py

```diff
@@ -52,6 +52,7 @@ def handle_connector_builder_request(
     if command == "resolve_manifest":
         return resolve_manifest(source)
     elif command == "test_read":
+        assert catalog is not None, "`test_read` requires a valid `ConfiguredAirbyteCatalog`, got None."
```

Comment:
This would be part of the API between the Connector Builder and the CDK. Is this what we expect?

Reply:
Yes, it ultimately gets wrapped in an AirbyteTraceMessage, which is handled by the connector builder server.